packages/config/src/scripts/dev-script.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-console */
+const inquirer = require('inquirer');
+const { resolve } = require('path');
+const { statSync } = require('fs');
+const { spawn } = require('child_process');
+const { logError } = require('./common');
+
+async function setEnv(cwd) {
+  return inquirer
+    .prompt([
+      {
+        type: 'list',
+        name: 'clouddotEnv',
+        message: 'Which platform environment you want to use?',
+        choices: ['stage', 'prod', 'qa', 'ci'],
```

Comment:
Do we really want to offer qa and ci?

Reply:
Yes, there are some still using it, however I will add a deprecated note to these options.